src/praisonai-agents/praisonaiagents/agent/tool_execution.py

```
@@ -846,38 +847,40 @@ async def check_pending_approvals(self) -> Dict[str, Any]:
+                                "status": "approved_and_executed",
+                                "tool_name": info["function_name"],
+                                "decision": decision,
+                                "result": tool_result,
+                            }
+                        else:
+                            results[tid] = {
+                                "status": "denied",
+                                "tool_name": info["function_name"],
+                                "decision": decision,
+                            }
+                    except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)